### PR TITLE
PR 666 => develop 

### PR DESCRIPTION
**Pull request by VladislavStavila**
# Jira Ticket:

Link to corresponding jira ticket:
https://pib-rocks.atlassian.net/jira/software/c/projects/PR/boards/3?selectedIssue=PR-666

# Checklist:

Necessary tasks before pull request can be merged:

-   [ ] The code has been reviewed and approved at least once
-   [ ] All necessary tests have been successful
-   [ ] The product owner reviewed and accepted the story

# Testing:

### Test acceptance criteria:

-   [ ] Dev Test

### Setup instructions:

-   [ ] Setup on a virtual machine was successful

### CODE REVIEW

File: src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.scss [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
 }
 
 input[type="checkbox"] {
     appearance: none;
-    color: currentColor;
+    color: $white-color;
     width: 1.15em;
     height: 1.15em;
-    border: 0.15em solid currentColor;
+    border: 0.15em solid $white-color;
```
Comment (nmsger-isento) [inline src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.scss:25]: Are they always white or were there instances where `currentColor` had a different value?

File: src/styles.scss [hunk 2]
```diff
@@ -1,3 +1,22 @@
+$pink-color: #e10072 !default;
+$blue-light: #0094df !default;
```
Comment (nmsger-isento) [inline src/styles.scss:2]: In any case I would also add a default `blue` color, so there's a base reference.

File: angular.json [hunk 3]
```diff
@@ -1,121 +1,146 @@
...
+						"polyfills": [
+							"zone.js"
+						],
+						"tsConfig": "tsconfig.app.json",
+						"assets": [
+							"src/favicon.ico",
+							"src/assets"
+						],
+						"styles": [
+							"@angular/material/prebuilt-themes/indigo-pink.css",
```
Comment (nmsger-isento) [inline angular.json:32]: Do our classes conflict with the prebuilt-themes (are we even using the material design?)

File: src/styles.scss (hunk 2 above)
Comment (VladislavStavila) [reply src/styles.scss]: $blue-light was changed to the $blue-color

File: src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.scss (hunk 1 above)
Comment (VladislavStavila) [reply src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.scss:25]: They are always white

**Review by nmsger-isento — APPROVED** [review]

**Pull request merged**